meta/meta.json

```diff
@@ -1,3 +1,3 @@
 {
-  "subnet_version": "4.8.1"
+  "subnet_version": "4.8.2"
```

Comment:
Better not to update meta.json for README updates as this will reset all validators